### ISSUE TITLE
aria-orientation and JAWS failure to enter forms mode reported fixed

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/a11y-research#95]
At this session: http://www.csun.edu/cod/conference/2018/sessions/index.php/public/presentations/view/584

VFO reported that this bug was fixed. We should verify and update our documentation accordingly. We encountered this issue in https://github.com/phetsims/a11y-research/issues
